docs/features/schema-registry.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Schema Registry'
+sidebar_position: 7
+keywords: [ 'schema', 'avro', 'confluent' ]
+---
+
+It is used internally in Conduit to store the schemas of records that are produced by connectors that generate **structured data** (encoded into [Avro format](https://avro.apache.org/)). The schema registry can either be embedded in a Go application or run as a standalone service exposing a REST API that's compatible with [Confluent's Schema registry](https://docs.confluent.io/current/schema-registry/develop/api.html).
```

Comment:
Nit: maybe starting the sentence with "The schema registry is used internally in Conduit..." sounds better?

Reply:
Btw, what exactly is meant by this " The schema registry can either be embedded in a Go application"? Conduit?

Reply:
@hariso yeah, I was thinking about showing off the ability to include this in a different service as a dependency, but maybe that defeats the purpose of Conduit itself...